includes/admin/views/html-admin-settings.php

```diff
@@ -81,8 +85,41 @@
 
         <div class="nav-tab-wrapper give-nav-tab-wrapper">
             <?php
+            /**
+             * Render Recurring Donations UTM link.
+             *
+             * @unreleased
+             */
             foreach ($tabs as $name => $label) {
-                echo '<a href="' . admin_url('edit.php?post_type=give_forms&page=' . self::$setting_filter_prefix . "&tab={$name}") . '" class="nav-tab ' . ($current_tab === $name ? 'nav-tab-active' : 'give-mobile-hidden') . '">' . $label . '</a>';
+                $target = $name === 'recurring' ? 'target="_blank" ' : false;
+                $urlPath = $name === 'recurring' ? 'https://docs.givewp.com/recurring-link' : admin_url(
+                    'edit.php?post_type=give_forms&page=' . self::$setting_filter_prefix . "&tab={$name}"
+                );
+                echo '<a ' . $target . 'href="' . $urlPath . '"' . ' class="nav-tab ' . ($current_tab === $name ? 'nav-tab-active' : 'give-mobile-hidden') . '">' . $label . '</a>';
+            }
+
+            /**
+             * Render Addon product recommendation link if Recurring Donations is active.
+             *
+             * @unreleased
+             */
+            if (class_exists('Give_Recurring')) {
```

Comment:
Don't we have a Utility to check if the addon is installed?

Reply:
Yes definitely do. Ty! https://github.com/impress-org/givewp/pull/6794/commits/50ab907c9c7e8f671625a4eeb32e884b0b01610b